bin/craft

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env php
+<?php
+/**
+ * Craft console bootstrap file
+ */
+
+if (!empty($_composer_autoload_path)) {
+    $basePath = dirname($_composer_autoload_path) . '/../';
```

Comment:
One above `vendor/` seems like a safe enough assumption. If you're directly requiring `craftcms/cms` and not using `craftcms/craft` your "project root" would still be one level above.

Locally, though, whenever I bring `./craft` in to a non-standard environment I usually use `getcwd()` as my base path. This seems very reliable for console/CLI applications. Won't make sense for web/FPM installs though but this is only run via console commands so that shouldn't matter.